index.html

```diff
@@ -7714,6 +7714,80 @@ <h3>Processing actions</h3>
  input JSON, such that the actions to be performed in a single <a>tick</a>
  are grouped together.
 
+<p>An <dfn>element origin</dfn> is a Map
+which [=map/exist|contains=] "<code>type</code>" and
+"<code>sharedId</code>", and for which the value of the entry with key
+"<code>type</code>" is "<code>node</code>".
+
+<p>To <dfn>get coordinates relative to an origin</dfn>
+given <var>source</var>, <var>x offset</var>, <var>y offset</var>,
+and <var>origin</var>:
+
+<ol class="algorithm">
+ <li><p>Run the substeps of the first matching value
+  of <var>origin</var>
+  <p>Assert: one of these conditions matches
+
+  <dl>
+   <dt>"<code>viewport</code>"
+   <dd>
+    <ol>
+     <li><p>Let <var>x</var> equal <var>x offset</var> and
+     <var>y</var> equal <var>y offset</var>.
+    </ol>
+   </dd>
+
+   <dt>"<code>pointer</code>"
+   <dd>
+    <p class="note">This is only possible if <var>source</var> is
+    a <a>pointer input source</a></p>
+
+    <ol>
+     <li><p>Let <var>start x</var> be equal to the <code>x</code>
+     property of <var>source</var>.
+
+     <li><p>Let <var>start y</var> be equal to the <code>y</code>
+     property of <var>source</var>.
+
+     <li><p>Let <var>x</var> equal <var>start x</var> +
+     <var>x offset</var> and <var>y</var> equal
+     <var>start y</var> + <var>y  offset</var>.
+    </ol>
+   </dd>
+   <dt>An <a>element origin</a>, or an object that <a>represents a web element</a></dt>
+   <dd>
+    <ol>
+     <li>
+      <p>If <var>origin</var> is an <a>element origin</a>:
+
+      <ol>
+       <li><p>Let <var>element</var> be the item in the <a>current browsing
+          context</a>’s <a>list of known elements</a> for which the <a>web
+          element reference</a> is equal to <var>origin</var>["<code>sharedId</code>"],
```

Comment:
Nothing in the current WebDriver classic spec actually uses `sharedId`. So if only classic is used any retrieved and serialized node will not be added to the `list of known elements` containing a `sharedId` property.

I assume that this PR is blocked by some other PR. You might want to mention that in the initial comment and probably we should add a new label to indicate that? Right now dependencies are not visible.

Reply:
The intent is that by getting the `sharedId` property from the object you're effectively extracting the web element reference identifier. Other parts of the spec shouldn't care that it came from a BiDi style `{sharedId: <id>}` object vs a class-style webelement reference object.